blocks/library/table/index.js

```diff
@@ -27,10 +28,29 @@ registerBlockType( 'core/table', {
 			type: 'array',
 			source: children( 'table' ),
 			default: [
```

Comment:
We might want to keep what we had before and provide transformation layer instead. I wanted to make it work.